sdk/cosmosdb/ci.yml

```diff
@@ -9,25 +8,24 @@ trigger:
   paths:
     include:
       - sdk/cosmosdb/
-
 pr:
   branches:
     include:
       - main
       - feature/*
       - release/*
       - hotfix/*
+    exclude:
+      - feature/v4
   paths:
     include:
       - sdk/cosmosdb/
-
 extends:
-  template: ../../eng/pipelines/templates/stages/cosmos-sdk-client.yml #NOTE: Non-standard template.
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```

Comment:
@praveenkuttappan @ramya-rao-a   Do you have any idea why cosmosdb is using a different template here ? Is it okay to change it into archetype-sdk-client.yml and enable the unit tests now?

Reply:
@jeremymeng You have the context here right?

Reply:
I remember cosmos config is different from other packages. It needs to setup the emulator to run tests.  @HarshaNalluru may also have more info.

Reply:
Yes, cosmos sdk uses the emulator instead of the live service for testing, and hence different template.

Reply:
@colawwj  changed it back.

Reply:
Just realize the mgmt plane test won't be triggerred in that template. how are we going to make it work @HarshaNalluru @jeremymeng

Reply:
@qiaozha if I understand correctly, we need to enable ci for cosmos mgmt plane using the standard template, while the cosmos dataplane keeps using the non-standard one?

Reply:
That sounds right, @jeremymeng.

Reply:
@praveenkuttappan any recommendations here?

Reply:
We will need two different pipeline under cosmos service in that case with different artifact set and triggering path.
1. One for cosmos package using emulator
2. another one for mgmt  package.

I do not prefer that option unless we have no other choice. why does cosmos mgmt needs different type of testing? Is it because it doesn't need emulator at all?

Reply:
I have submitted PR https://github.com/Azure/azure-sdk-for-js/pull/18659 to add new pipeline for cosmos mgmt package.

Reply:
and this has created pipeline https://dev.azure.com/azure-sdk/public/_build?definitionId=3311